packages/voice/src/models/v1/ace-request/ace-request.ts

```diff
@@ -13,7 +13,7 @@ import { AceRequestAmd } from '../ace-request-amd';
 export interface AceRequest {
 
   /** Must have the value `ace`. */
-  event?: string;
+  event?: 'ace';
```

Comment:
As it has value, remove optionality?
```suggestion
  event: 'ace';
```